vaadin-grid-flow-parent/vaadin-grid-flow/src/main/java/com/vaadin/flow/component/grid/GridMultiSelectionModel.java

```diff
@@ -58,7 +62,7 @@
         HIDDEN,
 
         /**
-         * By default select all checkbox depends on the grid's dataprovider.
+         * By default, select all checkbox depends on the grid's data provider.
          * <ul>
          * <li>Visible, if the data provider is in-memory</li>
          * <li>Hidden, if the data provider is NOT in-memory (lazy)</li>
```

Comment:
Maybe the `@see` below is not anymore helpful and could be removed or pointing to how the data is set either as inmemory or lazy

Reply:
Removed. I didn't place the `setItems` references here, because there are several of them. IMO it would look ugly if I place it there.